integration/Gradient-Train-Job.py

```diff
@@ -18,6 +18,8 @@
 # MAGIC  * The Gradient Webhook has been configured
 # MAGIC  * The Databricks Job has been Gradient enabled
 # MAGIC
+# MAGIC  When bypassing the Gradient Webhook with AWS, the cluster attached to this notebook must have an instance_arn with describe_instances and describe_volumes permissions. When bypassing the Gradient Webhook with Azure, the following environment variables must be set with the correct values: "AZURE_TENANT_ID", "AZURE_SUBSCRIPTION_ID", "AZURE_CLIENT_SECRET", "AZURE_CLIENT_ID"
```

Comment:
Just added this in case it's unclear what customers would need to do to bypass the webhook. I'm fine with taking it out if bypassing the webhook is supposed to be a complete edge case